plutus-tx-plugin/src/Language/PlutusTx/Plugin.hs

```diff
@@ -265,7 +265,7 @@ convertExpr opts locStr origE resType = do
               (plcP::PLCProgram) <- void <$> (flip runReaderT PIR.NoProvenance $ PIR.compileProgram pirP)
               when (poDoTypecheck opts) $ do
                   annotated <- PLC.annotateProgram plcP
-                  void $ PLC.typecheckProgram (PLC.TypeCheckCfg 1000 (PLC.TypeConfig True stringBuiltinTypes)) annotated
+                  void $ PLC.typecheckProgram (PLC.TypeConfig True mempty (Just 1000)) annotated
```

Comment:
Looks like this conflict got fixed wrong: should be stringBuiltinTypes instead of mempty.

Reply:
Ah, my mistake.